src/internal/_reduced.js

```diff
@@ -1,3 +1,7 @@
 module.exports = function(x) {
-  return x && x.__transducers_reduced__ ? x : {value: x, __transducers_reduced__: true};
+  return x && x['@@transducer/reduced'] ? x :
```

Comment:
ok. i'll know that i have arrived when i get *no* review comments from you :smile: